### PR TITLE
Add multiple issue templates

### LINKED ISSUE #9344: adopt github issue templates
see: https://github.com/kubernetes/org/tree/master/.github/ISSUE_TEMPLATE

we can have a lovely UI with different templates for different kinds of issues and make it easier and simpler for contributors to file bugs / feature requests etc

image

/kind enhancement
/area github
cc @cblecker @cjwagner @spiffxp

### PR DESCRIPTION
**Pull request by shubheksha**
Fixes #9344 
/cc @BenTheElder @cjwagner

### CODE REVIEW

**Comment by shubheksha** [comment]
ping @cjwagner @BenTheElder

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+<!-- DO NOT EDIT BELOW THIS LINE -->
+/king bug
```
Comment (cjwagner) [inline .github/ISSUE_TEMPLATE/bug-report.md]: s/king/kind/
:)

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the test-infra project
```
Comment (cjwagner) [inline .github/ISSUE_TEMPLATE/enhancement.md]: nit: There isn't really a "test-infra project" this repo is more of a collection of tools.
"Suggest an enhancement to the test-infra tooling." might be more appropriate.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
```
Comment (cjwagner) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Please ask for links to example occurrences as well.

**Comment by shubheksha** [comment]
Squashed the commits, cancelling the hold.
/hold cancel

**Pull request merged**